core/src/autogluon/core/pseudolabeling/pseudolabeling.py

```diff
@@ -9,12 +9,14 @@
 
 def sample_bins_uniformly(y_pred_proba: pd.DataFrame, df_indexes):
     """
-    Takes predictive probs and finds the minimum class count then samples that
-    from every class with rows with index in df_indexes
+    Takes predictive probabilities from y_pred_proba and finds the minimum
+    class count then samples minimum class count from every class with
+    rows with indexes in df_indexes
 
     Parameters:
     y_pred_proba: Predicted probabilities for multi-class problem
-    df_indexes: The indices that should be taken into consideration when sampling evenly
+    df_indexes: The indices of y_pred_proba should be taken into consideration when
+        sampling evenly
```

Comment:
purpose of this could be more clear, eg. these indices may correspond to those used in the dataframe containing the raw data.

Reply:
added in more comments